src/coprocessor/codec/data_type/vector.rs

```diff
@@ -601,7 +599,7 @@ impl VectorValue {
                     }
                     Some(ref val) => {
                         output.push(datum::DURATION_FLAG);
```

Comment:
use `write_u8`?

Reply:
done

Reply:
and there is a FIXME `    // FIXME: Use BufferWriter.` you mean to change the function argument ` output: &mut Vec<u8> ` to accept a BufferWriter or make the output use the BufferWriter trait?